### PR TITLE
minor documentation improvements

### PR DESCRIPTION
**Pull request by thefourtheye**

### CODE REVIEW

File: docs/java-api/client.asciidoc [hunk 1]
```diff
@@ -59,18 +59,18 @@ Client client = TransportClient.builder().settings(settings).build();
...
 allows it to dynamically add new hosts and remove old ones.
-When sniffing is enabled the transport client will connect to the nodes in its
-internal node list, which is built via calls to addTransportAddress.
+When sniffing is enabled, the transport client will connect to the nodes in its
+internal node list, which is built via calls to `addTransportAddress`.
 After this, the client will call the internal cluster state API on those nodes
 to discover available data nodes. The internal node list of the client will
 be replaced with those data nodes only. This list is refreshed every five seconds by default.
-Note that the IP addresses the sniffer connects to are the ones declared as the 'publish'
+Note that the IP addresses, the sniffer connects to, are the ones declared as the 'publish'
```
Comment (clintongormley) [inline docs/java-api/client.asciidoc]: these commas are incorrect.  better to say "Note that the IP addresses that the sniffer connects to are"

File: docs/java-api/client.asciidoc [hunk 2]
```diff
@@ -59,18 +59,18 @@ Client client = TransportClient.builder().settings(settings).build();
...
 address in those node's elasticsearch config.
 
-Keep in mind that list might possibly not include the original node it connected to
+Keep in mind that list might possibly not include the original node it connected to,
```
Comment (clintongormley) [inline docs/java-api/client.asciidoc]: Keep in mind that THE list, and delete the added comma

**Comment by clintongormley** [comment]
Hi @thefourtheye 

Thanks for the PR.  I've added some corrections.  If you could update the PR with the changes then I'll be happy to merge it. 

thanks

**Comment by thefourtheye** [comment]
@clintongormley Thanks for the review. I updated the PR now.

**Comment by clintongormley** [comment]
thanks @thefourtheye - merged

**Pull request merged**